backend-tests/tests/test_useradm_enterprise.py

```diff
@@ -71,6 +71,12 @@ def _login(self, user, totp=None):
         r = uadm.call("POST", useradm.URL_LOGIN, auth=(user.name, user.pwd), body=body)
         return r
 
+    def _verify(self, utoken, totp):
```

Comment:
This method is only used once, consider removing it

Reply:
I am sorry, but please lets keep it, it is logical and cleaner this way.

Reply:
:+1: